### PR TITLE
Add license section for brevity.

### LINKED ISSUE #870: Please add copyright and license notice to README.md
The project contains an Apache 2.0 license file, however, it does not follow the directions in the appendix of the license: https://github.com/minio/minio-go/blob/master/LICENSE#L179-L202

For clarity, the README should "apply" the license as directed, including copyright information. Since the code was written primarily by the Minio team, it's probably easiest to indicate Minio, Inc as the copyright owner.

### PR DESCRIPTION
**Pull request by harshavardhana**
Sort of redundant but add the section anyways to
be more forthcoming. Fixes #870

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -245,3 +245,5 @@ The full API Reference is available here.
 [![Build Status](https://travis-ci.org/minio/minio-go.svg)](https://travis-ci.org/minio/minio-go)
 [![Build status](https://ci.appveyor.com/api/projects/status/1d05e6nvxcelmrak?svg=true)](https://ci.appveyor.com/project/harshavardhana/minio-go)
 
+## License
+This SDK is distributed under the [Apache License, Version 2.0](http://www.apache.org/licenses/LICENSE-2.0), see LICENSE for more information.
```
Comment (nitisht) [inline README.md]: May be add a link to the LICENSE doc?

File: README.md [hunk 2]
```diff
@@ -1,19 +1,7 @@
-# Minio Go Client SDK for Amazon S3 Compatible Cloud Storage [![Slack](https://slack.minio.io/slack?type=svg)](https://slack.minio.io) [![Sourcegraph](https://sourcegraph.com/github.com/minio/minio-go/-/badge.svg)](https://sourcegraph.com/github.com/minio/minio-go?badge)
+# Minio Go Client SDK for Amazon S3 Compatible Cloud Storage [![Slack](https://slack.minio.io/slack?type=svg)](https://slack.minio.io) [![Sourcegraph](https://sourcegraph.com/github.com/minio/minio-go/-/badge.svg)](https://sourcegraph.com/github.com/minio/minio-go?badge) [![Apache V2 License](http://img.shields.io/badge/license-Apache%20V2-blue.svg)](https://github.com/minio/minio-go/blob/master/LICENSE)
 
 The Minio Go Client SDK provides simple APIs to access any Amazon S3 compatible object storage.
 
-**Supported cloud storage providers:**
```
[the remark this answers is not in the document]
Comment (harshavardhana) [reply README.md]: Not needed we support pretty much all S3 providers and most implement v4 now. Since other SDKs don't have it we don't need it here..

**Comment by deekoder** [comment]
Adding a reviewer to make a release today.

File: pkg/credentials/static_test.go [hunk 3]
```diff
@@ -1,6 +1,6 @@
 /*
- * Minio Go Library for Amazon S3 Compatible Cloud Storage
- * (C) 2017 Minio, Inc.
+ * Copyright
+ *  2017 Minio, Inc.
```
Comment (krishnasrinivas) [inline pkg/credentials/static_test.go]: doesnt look right, should be:
Minio Go Library for Amazon S3 Compatible Cloud Storage
Copyright 2017 Minio, Inc.

File: functional_tests.go [hunk 4]
```diff
@@ -6537,7 +6538,7 @@ func main() {
 		testComposeObjectErrorCases()
 		testCompose10KSources()
 		testUserMetadataCopying()
-		testEncryptedCopyObject()
+		// testEncryptedCopyObject()
```
Comment (krishnasrinivas) [inline functional_tests.go]: testEncryptedCopyObjectV2 testEncryptedCopyObject commented out, intentional?

Comment (harshavardhana) [reply functional_tests.go]: Yes CopyObject is not supported yet with SSE-C commenting it out opening a new issue to re-enable it with server feature in place.

Comment (deekoder) [reply functional_tests.go]: As per discussion, we will return "Not Implemented" from the server instead of commenting this test out here.

**Pull request merged**